pyemojify/cli.py

```diff
@@ -1,19 +1,35 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+"""
+Usage:
+    pyemojify [<text>]
+    pyemojify (-h | --help)
+    pyemojify (-v | --version)
+
+Options:
+    <filename>    The input text.
+    -h --help     Show this help.
+    -v --version  Show version information.
+"""
+
 from __future__ import absolute_import
 
-import click
+import sys
+
+from docopt import docopt
 
 from pyemojify import __version__
 from pyemojify.emojify import emojify
 
+if __name__ == "__main__":
+    argv = docopt(__doc__)
```

Comment:
You can use:

    argv = docopt(__doc__, version=__version__)

so `docopt` will handle the version info for you.